bitnami/nginx/README.md

```diff
@@ -397,7 +398,7 @@ Affected values:
 
 This major release no longer uses the bitnami/nginx-ldap-auth-daemon container as a dependency since its upstream project is not actively maintained.
 
-*2022-04-12 edit*:
+_2022-04-12 edit_:
```

Comment:
I guess this is caused by some formatter you have installed in your system. Could you please remove these changes from your commit?